etl/config.py

```
@@ -1,173 +1,12 @@
-
-    cfg["sources"] = norm
-    return norm
+"""Config utilities - backward compatibility module.
+
+This module provides backward compatibility for config utilities.
+The actual implementation is in etl.utils.config.
+"""
+
+# Import all config utilities for backward compatibility
+try:
+    from .utils.config import *
```

Ruff (0.12.2)

`from .utils.config import *` used; unable to detect undefined names

(F403)